airflow/operators/trigger_dagrun.py

```diff
@@ -167,6 +167,9 @@ def execute(self, context: Dict):
             else:
                 raise e
 
+        if not dag_run:
```

Comment:
Need to validate if this is OK to do

Reply:
I believe  it's good, though error message could be more descriptive. "Could not find dag_run to trigger" ?.  What do others think ?